tests/default/data/dibi.php

```diff
@@ -10,13 +10,15 @@ class Foo
     public function fetch(\Dibi\Connection $connection)
     {
         $row = $connection->fetch('SELECT email, adaid FROM ada where adaid = %i', 1);
-        assertType('array{email: string, adaid: int<-32768, 32767>}|null', $row);
+        assertType('Dibi\Row<array{email: string, adaid: int<-32768, 32767>}>|null', $row);
+        assertType('int<-32768, 32767>', $row->adaid);
+        assertType('int<-32768, 32767>', $row['adaid']);
```

Comment:
the array access should be covered by the stubs generic type and `\ArrayAccess` with said type